test/JDBC/expected/BABEL-745-vu-prepare.out

```diff
@@ -0,0 +1,67 @@
+-- Create SalesData table
+CREATE TABLE SalesData (
+  Product varchar(50),
+  SalesAmount decimal(10,2)
```

Comment:
your function signature is receiving float value. 
So please also test with float value